Presentation.Web/app/components/it-contract/tabs/it-contract-tab-main.controller.ts

```diff
@@ -278,11 +280,11 @@
                     else if (value == null) {
                         //made to prevent error message on empty value i.e. open close datepicker
                     }
-                    else if (!date.isValid() || isNaN(date.valueOf()) || date.year() < 1000 || date.year() > 2099) {
-                        notify.addErrorMessage("Den indtastede dato er ugyldig.");
+                    else if (Kitos.Helpers.DateValidationHelper.checkIfDateIsInvalid(date, notify)) {
+                        return;
                     }
-                    else if (fromDate >= endDate) {
-                        notify.addErrorMessage("Den indtastede slutdato er før startdatoen.");
+                    else if (Kitos.Helpers.DateValidationHelper.checkIfStartDateIsGreaterThanEndDate(fromDate, endDate, notify, "startdatoen", "slutdato")) {
```

Comment:
are the names correct? I copied them from the previous error message

Reply:
use names from the UI

Reply:
and add \"\" to them